app/containers/Service/Texts/Liturgies/Zlatoust.js

```diff
@@ -2882,7 +2882,7 @@ const Zlatoust = ({ lang, date }) => {
                         Как полнота Закона и Пророков, Христе Боже наш, <br />
                     </span>
                     <span className="_-ВЫДЕЛЕНИЯ_ЧЁРНЫЙ">
-                        Ты исполнил весь замысел
+                        Ты исполнил весь замысел 
```

Comment:
Так в реакте работать не будет
```suggestion
                        Ты исполнил весь замысел{' '}
```